core/trino-main/src/main/java/io/trino/metadata/FunctionRegistrations.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.metadata;
+
+import io.trino.operator.scalar.JsonExistsFunction;
+import io.trino.operator.scalar.JsonQueryFunction;
+import io.trino.operator.scalar.JsonValueFunction;
+import io.trino.spi.type.TypeManager;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import static java.util.Objects.requireNonNull;
+
+public class FunctionRegistrations
```

Comment:
do we need this?
What is the benefit over adding
```java
    @ProvidesIntoSet
    @Singleton
    public static FunctionBundle jsonFunctionBundle(TypeManager typeManager, FunctionManager functionManager, Metadata metadata)
    {
        return new InternalFunctionBundle(
                new JsonExistsFunction(functionManager, metadata, typeManager),
                new JsonValueFunction(functionManager, metadata, typeManager),
                new JsonQueryFunction(functionManager, metadata, typeManager));
    }
``` 

to `ServerMainModule`.
Or maybe even passing missing dependencies to `SystemFunctionBundle` and register json functions there.

Reply:
Thanks for the hint. Added as another bundle.